libs/runtime/ebpf_crypto_hash.c

```diff
@@ -122,3 +123,62 @@ ebpf_cryptographic_hash_get_hash_length(_In_ const ebpf_cryptographic_hash_t* ha
     }
     return EBPF_SUCCESS;
 }
+
+_Must_inspect_result_ ebpf_result_t
+ebpf_cryptographic_hash_of_file(
```

Comment:
Should have a verb in the function name.
Maybe `ebpf_get_cryptographic_hash_of_file()` or `ebpf_hash_file_contents()`